### DISCUSSION TITLE
开源面对面官网域名投票🗳️

### DISCUSSION THREAD

**Discussion by MikeyWei** [opensource-f2f/episode#146] [Polls]
在[声网](https://www.agora.io)的赞助下，我们可以推进官网的搭建了。
当前要做的第一件事：选择一个域名，然后购买一个云主机。

本着公开（Open）的原则，根据咱们的项目名，发起本次域名投票（不含域名后缀）。

投票截至北京时间2022年7月25日，欢迎大家踊跃投票。

由于需要备案等后续操作，以及经费限制等因素，不能在大陆地区备案的后缀将不在考虑范围，是否需要对域名的后缀选择进行投票也欢迎大家跟贴反馈。

**Comment by kang8** [comment]
两个都想要怎么办 😂

**Reply by MikeyWei** [reply]
> 两个都想要怎么办 😂

等我们有了钱……都买下

**Reply by LinuxSuRen** [reply]
不知道我理解的对不对，一般买多个域名是为了避免其他人（组织）抢注了。例如：百度可能会把 `baidu` 相关的域名全注册下来。我们暂时可能没有这种诉求。当然，有同学愿意帮忙注册下来也很好。

**Reply by MikeyWei** [reply]
对，大多数是这个诉求。也有少数”囤“域名是为了待价而沽赚差价的。当然，我们没这个需求，看现在的投票结果，osf2f 将胜出。

**Comment by S1ngS1ng** [comment]
OpenSourceF2F 会不会好些

**Reply by miyaliu666** [reply]
域名好像都是小写？大写用作 logo 里的英文字样吧

**Comment by juzhiyuan** [comment]
后缀选择哪个呢？

**Reply by MikeyWei** [reply]
后缀更多地是看价格了，并且要能在大陆备案的。

**Comment by shinyzhu** [comment]
越短越好。大家都喜欢缩写～

**Comment by MikeyWei** [comment]
官网已上线，感谢各位。
